client/client-lib/src/lib.rs

```diff
@@ -654,15 +662,18 @@ impl<T: AsRef<ClientConfig> + Clone> Client<T> {
             .await;
 
         let mut dbtx = self.context.db.begin_transaction().await;
-        let mut all_notes = TieredMulti::<SpendableNote>::default();
+        let mut notes_to_reissue = TieredMulti::<SpendableNote>::default();
         for result in stream.collect::<Vec<_>>().await {
             let (key, notes) = result?;
-            all_notes.extend(notes);
+            notes_to_reissue.extend(notes);
             dbtx.remove_entry(&key).await.expect("DB Error");
         }
         dbtx.commit_tx().await.expect("DB Error");
 
-        self.reissue(all_notes, rng).await
+        debug!(target: LOG_WALLET, notes_to_reissue = ?notes_to_reissue.summary(), total = %notes_to_reissue.total_amount());
+        trace!(target: LOG_WALLET, ?notes_to_reissue, "foo");
```

Comment:
Accidentally merged?

Reply:
Nope. I think these are useful, and helped me to debug this.